tests/auth_tests/test_hashers.py

```diff
@@ -457,6 +457,12 @@ def test_argon2(self):
         self.assertTrue(is_password_usable(blank_encoded))
         self.assertTrue(check_password('', blank_encoded))
         self.assertFalse(check_password(' ', blank_encoded))
+        # Old hashes without version attribute
+        encoded = ('argon2$argon2i$m=8,t=1,p=1$c29tZXNhbHQ$gwQOXSNhxiOxPOA0+'
+                   'PY10P9QFO4NAYysnqRt1GSQLE55m+2GYDt9FEjPMHhP2Cuf0nOEXXMoc'
+                   'VrsJAtNSsKyfg')
+        self.assertTrue(check_password('secret', encoded))
+        self.assertFalse(check_password('wrong', encoded))
 
     def test_argon2_upgrade(self):
```

Comment:
That used to be there when I still used the version attribute...

Ok, I'll create a separate test.